pom.xml

```diff
@@ -18,6 +18,22 @@
 		<spring-cloud.version>2022.0.3</spring-cloud.version>
 	</properties>
 	<dependencies>
+		<dependency>
```

Comment:
All changes made in the POM file already exist in the main branch. Please, keep your branch synced with master by often pulling from the mater branch